Fabric.Authorization.API/scripts/AzureAD-Migration.ps1

```diff
@@ -0,0 +1,192 @@
+#Requires -RunAsAdministrator
+#Requires -Version 5.1
+#Requires -Modules PowerShellGet, PackageManagement
+
+param(
+    [PSCredential] $credential,
```

Comment:
Consider adding [CmdletBinding()] - it brings in a bunch of "standard" powershell parameters like "-Verbose" and others that are helpful.